porcupine/tabs.py

```diff
@@ -432,9 +443,9 @@ def _ask_encoding(path: pathlib.Path, encoding_that_didnt_work: str) -> str | No
     ).pack(fill="x", padx=10, pady=10)
 
     var = tkinter.StringVar()
-    entry = ttk.Entry(big_frame, textvariable=var)
-    entry.pack(pady=50)
-    entry.insert(0, encoding_that_didnt_work)  # type: ignore[no-untyped-call]
+    combobox = ttk.Combobox(big_frame, values=sorted(list_of_encodings), textvariable=var)
```

Comment:
Maybe the whole `textvariable` should be deleted, since this uses `combobox.get()` anyway.

Reply:
I kept the `textvariable` so that if the user types an unknown encoding, it checks and disables the Ok button.